src/entity/Wallet.ts

```diff
@@ -35,6 +36,18 @@ export class Wallet extends BaseEntity {
 
     private unlocked_private_key: string;
 
+    get asyncEvmAddress(): Promise<string> {
+        // Potential local caching opportunity here
```

Comment:
Not 100% sure what this means. If it's a TODO, maybe flag it as such so it's easier to find in the future?